sklearn/metrics/_regression.py

```diff
@@ -188,6 +190,78 @@ def mean_absolute_error(y_true, y_pred,
     return np.average(output_errors, weights=multioutput)
 
 
+def mean_absolute_percentage_error(y_true, y_pred,
+                                   sample_weight=None,
+                                   multioutput='uniform_average'):
+    """Mean absolute percentage error regression loss
+
```

Comment:
Note here that we do not represent the output as a percentage: 1, rather than 100, indicates that the difference between y_pred and y_true is equal to y_true.

Reply:
Done